qml/Launcher/Drawer.qml

```diff
@@ -292,6 +292,7 @@ FocusScope {
                 readonly property bool focused: index === GridView.view.currentIndex && GridView.view.activeFocus
 
                 onClicked: root.applicationSelected(model.appId)
+                onPressAndHold: Qt.openUrlExternally("https://open-store.io/app/" + model.appId.split("_")[0] + "/")
```

Comment:
I'm not sure we should just open a link in another app like this, directly from `pressAndHold`.  I think we need more integration between Launcher and Drawer in terms of what we do when interacting with app icons.

Reply:
Agreed. But maybe this could be a quick-and-dirty fix for now to make the drawer less broken. 100% with you that drawer and launcher should be more integrated, but given that it takes some refactoring to pull out the quicklistmenu and we don't know what direction UAL will take, i'd put a pin in that. And when the time comes, it's just one line of code to remove.

Reply:
Hmm. This will also open the store in a broken manner I think, for apps which are not clicks (from debs or libertine), or for locally installed clicks which aren't in the store.